sql/moz-fx-data-shared-prod/telemetry/clients_daily/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+---
+friendly_name: Clients Daily
+description: >-
+  A daily aggregate of main and event pings from each Firefox
+  desktop client.
+
+  Clustering fields: `normalized_channel`, `sample_id`
+
+  See also: `clients_last_seen`, `clients_first_seen`
```

Comment:
This seems out of sync with the description in the derived dataset: 

https://github.com/mozilla/bigquery-etl/blob/master/sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/metadata.yaml

It would be nice if we only had a single source of truth for these descriptions-- I wonder if that means that we should make it more clear that the queries in telemetry_derived are "behind the scenes" in their metadata and keep any notes in there implementation-specific.

Reply:
Very good point. Will update the table descriptions to that effect.

Reply:
I've simplified the descriptions for the underlying tables, and they now reference the user-facing view for more details.